### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by vikivivi**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -4248,6 +4249,7 @@ certifciate->certificate
 certifciated->certificated
 certifciates->certificates
 certifciation->certification
+certifiate->certificate
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:4252]: Can you add the -ed, -ion, -es and -ing variants of this typo please.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -6916,6 +6918,7 @@ creatre->create
 creatred->created
 creats->creates
 credate->created
+credetial->credential
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:6921]: Can you add the plural version of this typo too please.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -10720,6 +10723,7 @@ exibition->exhibition
 exibitions->exhibitions
 exicting->exciting
 exinct->extinct
+exipration->expiration
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:10726]: Can you add these too:
```suggestion
exipration->expiration
exipre->expire
exipred->expired
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -24850,6 +24854,7 @@ telocom->telecom
 telphony->telephony
 temaplate->template
 temaplates->templates
+temeprature->temperature
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:24857]: Can you add the plural please.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:4252]:
```suggestion
certifiate->certificate
certifiated->certificated
certifiation->certification
certifiations->certifications
certifiates->certificates
certifiating->certificating
```

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:24865]:
```suggestion
temeprature->temperature
temepratures->temperatures
```

**Comment by vikivivi** [comment]
Sorry. Too messy. Withdrawal of pull request.

**Comment by peternewman** [comment]
What's too messy about this @vikivivi , from a quick look it would still be fine to merge.

**Pull request closed without merging**